src/test/java/com/answerdigital/benhession/academy/answerkingweek2/controllers/ItemControllerTest.java

```diff
@@ -21,19 +21,15 @@
 import org.springframework.test.web.servlet.ResultActions;
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
-
 import java.math.BigDecimal;
 import java.util.List;
-
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.BDDMockito.given;
-import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.when;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
```

Comment:
Is it possible for us to replace this with the individual items we're importing?

Reply:
I've spoken with Oleksii about this and we agreed that changing * imports on tests will be done as part of a separate ticket as checkstyle isn't looking for this right now.